statistics/handle/handle.go

```diff
@@ -524,10 +538,17 @@ func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg
 	if err != nil {
 		return
 	}
-	data, err := statistics.EncodeCMSketch(cms)
+	data, topNData, err := statistics.EncodeCMSketch(cms)
 	if err != nil {
 		return
 	}
+	for i, v := range topNData {
+		insertSQL := fmt.Sprintf("replace into mysql.stats_topnstore (table_id, is_index, hist_id, value_id, content) values (%d, %d, %d, %d, X'%X')", tableID, isIndex, hg.ID, i, v)
+		_, err1 := exec.Execute(ctx, insertSQL)
+		if err1 != nil {
```

Comment:
`func (h *Handle) SaveStatsToStorage(tableID int64, count int64, isIndex int, hg *statistics.Histogram, cms *statistics.CMSketch, isAnalyzed int64) (err error)`

`err1` cannot be returned implicitly. Please `return err1`.
or why you use `err1` here instead of `err`?